### ISSUE TITLE
Explicit labeling for outside mfg

### ISSUE THREAD

**Issue by cush4x** [OpenAgInitiative/openag-mechanical#1]
Site:
https://wiki.openag.media.mit.edu/pfc_edu_3.0#getting_pfc_edu_parts_manufactured
speaks to having outside manufacturing of parts.

It is not clear what files a vendor would want/need to make a quote. Please consider creating or labeling a folder to do this.

**Comment by Ippako** [comment]
I agree. The file organization is not useful for actually getting one made/built and I'm familiar with many CAD systems. I would recommend putting together a PDF packet of all the unique parts and have a few assembly drawings that explain quantities and show exploded views.
Expecting the average "EDU" person to be able to sort this out with Solidworks is just too much and will end up resulting in disinterest.

**Comment by cush4x** [comment]
Yes, I have decided not to pursue this as the “overhead” is just too great. It needs to be more accessible to those of us with limited time/energy.
